db/migrate/20230217032241_create_investments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateInvestments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :investments do |t|
+      t.string :name
+      t.float :amount
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] I think the reference here should AuthorId and it should be an alias to the User table. If you remember the Blog App project you did where you had AuthorId referencing through the User as reference, I think you need to apply same knowedge here.